test/blueprints/Do.Test.Blueprints.Service/Authentication/Authentication.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Do.Test;
+
+public class Authentication(TimeProvider _timeProvider)
+{
+    public DateTime GetTime() => _timeProvider.GetNow();
```

Comment:
`GetTime` method was originally put to test dependency injection, so it is to see if `_timeProvider` was injected properly. move this method to another feature, i don't know, `Core` or `Business`or something.